subscription-engine/kafka-ws/app/services/TicketService.js

```diff
@@ -34,9 +34,9 @@ function generateTicket(token) {
 /**
  * Verify the integrity and authenticity of the token.
```

Comment:
Shouldn't be better to use the third person?

Reply:
I always forget that